tests/sqllogictests/suites/mode/standalone/explain/09_0039_target_build_merge_into_standalone.test

```diff
@@ -316,5 +316,76 @@ select * from target_build_optimization order by a,b,c;
 11 b11 c_11
 12 b12 c_12
 
+## add explain analyze
+statement ok
+create table t1(a int);
+
+statement ok
+insert into t1 values(1),(1);
+
+statement ok
+insert into t1 values(2);
+
+statement ok
+create table t2(a int);
+
+statement ok
+insert into t2 values(2);
+
+query T
+select count(*) from fuse_block('default','t1');
+----
+1
```

Comment:
CI shows the test has wrong result

Reply:
I think we should remove the test right?cc @xudong963